boefjes/boefjes/plugins/kat_sec_txt/normalize.py

```diff
@@ -0,0 +1,70 @@
+import json
+from typing import Iterable, Union
+
+from boefjes.job_models import NormalizerMeta
+from octopoes.models import OOI, Reference
+from octopoes.models.ooi.findings import Finding, KATFindingType
+
+
+def run(normalizer_meta: NormalizerMeta, raw: Union[bytes, str]) -> Iterable[OOI]:
+    results = json.loads(raw)
+    boefje_meta = normalizer_meta.raw_data.boefje_meta
+    ooi_ref = Reference.from_str(boefje_meta.input_ooi)
+
+    invalid = set(
```

Comment:
nit: you can use `{"location", "invalid_cert", ...}` instead of `set(["location", "invalid_cert", ...])` as the syntax to define sets.